code/modules/mob/living/simple_animal/abnormality/waw/judgement_bird.dm

```diff
@@ -183,3 +191,64 @@
 		patrol_path = get_path_to(src, target_turf, /turf/proc/Distance_cardinal, 0, 200)
 		return
 	return ..()
+
+//On-kill visual effect
+/obj/structure/jbird_noose
+	name = "feathery noose"
+	desc = "A structure found in the black forest."
+	icon = 'ModularTegustation/Teguicons/48x64.dmi'
+	icon_state = "noose"
+	pixel_x = -8
+	base_pixel_x = -8
+	max_integrity = 60
+	buckle_lying = 0
+	density = FALSE
+	anchored = TRUE
+	can_buckle = TRUE
+
+/obj/structure/jbird_noose/attack_hand(mob/user)
+	if(has_buckled_mobs())
+		for(var/mob/living/L in buckled_mobs)
+			user_unbuckle_mob(L, user)
+	else
+		..()
```

Comment:
```suggestion
	if(!has_buckled_mobs())
		return ..()
	for(var/mob/living/L in buckled_mobs)
		user_unbuckle_mob(L, user)
	return
```

Early return is good practice, this also makes it return the value of it's parent proc which should preserve any functionality structures might have with attack_hand if there's no buckled mob, also a good practice (unless you're specifically wanting to avoid parent code)